index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- <div class="d-flex flex-center"> -->
+        <header class="header d-flex flex-center">
+            <a href="#">Alejandro Ramos</a>
+            <a class="burger" href="#"><img src="images/IconMenu.png" alt="burger button"></a>   
+        </header>
+    <!-- </div> -->
```

Comment:
While improving the header as suggested you appear to have introduced some commented code. Please remove the commented code as it makes your code more readable.

Reply:
Thanks, Ted, I must be more careful 😅